libs/core/kiln_ai/adapters/extractors/extractor_runner.py

```
@@ -0,0 +1,110 @@
+            extractor_factory = extractor_adapter_from_type(
+                job.extractor_config.extractor_type,
+            )
+            extractor = extractor_factory(job.extractor_config)
+            if not isinstance(extractor, BaseExtractor):
+                raise ValueError("Not able to create extractor from extractor config")
+
+            if job.item.path is None:
+                raise ValueError("Document path is not set")
+
+            output = extractor.extract(
+                # FIXME: needing to resolve the path here is not ideal; should be done
```

Pylint (3.3.7)

[warning] FIXME: needing to resolve the path here is not ideal; should be done

(W0511)